backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8-alpine3.10
+
+RUN apk add --no-cache --virtual .build-deps gcc libc-dev make \
+    && pip install pipenv
+
+COPY Pipfile Pipfile
+COPY Pipfile.lock Pipfile.lock
+
+RUN pipenv install --dev
+
+COPY src /app
```

Comment:
Remember that default working directory for a container is `/` unless changed. When dealing with relative paths it is good to set the `WORKDIR app` so that you won't have to deal with strange bugs later on.

Reply:
I've added WORKDIR in 15902f0.